.github/workflows/selenium-test.yaml

```diff
@@ -191,4 +192,4 @@ jobs:
 
       - name: Cleanup Work Directory
         uses: AutoModality/action-clean@v1
-        if: always()
\ No newline at end of file
+        if: always()
```

Comment:
this isn't a change, not sure why it's showing up as one

https://github.com/m9sweeper/m9sweeper/blob/main/.github/workflows/selenium-test.yaml#L194

Reply:
You removed the newline at the end of the file